src/components/HomeHeader.tsx

```diff
@@ -0,0 +1,25 @@
+import Subtitle from './Subtitle';
+import LocationSelect from './LocationSelect';
+import { Location } from '../types/sparkeats';
+
+type HomeHeaderProps = {
+  locations: Location[];
+};
+
+function HomeHeader({ locations }: HomeHeaderProps) {
+  return (
+    <div className="home-header">
+      <Subtitle />
+
+      <LocationSelect locations={locations} />
+
+      <div className="home-header__button">
+        <a className="button__header-primary" href="/places/new">
```

Comment:
Can we replace this with a [React Router Link](https://reactrouter.com/en/main/components/link)?